content/voies-lyonnaises/ligne-2.json

```diff
@@ -67,7 +67,8 @@
         "id": "FontaineBarbe-commun",
         "line": 2,
         "name": "Île barbe - Pont Leclerc",
-        "status": "planned"
+        "status": "planned",
+        "link": "/voie-lyonnaise-2#fontaines-sur-saône-à-lîle-barbe"
```

Comment:
bien curieux de savoir comment tu arrives à ce beau lien ?

Quand tu cliques sur l'anchre, ça donne un truc comme ça :
`/voie-lyonnaise-2#fontaines-sur-sa%C3%B4ne-%C3%A0-l%C3%AEle-barbe` à cause des caractères spéciaux.

Tu les as édités à la main pour rendre plus propre ?

Reply:
En l'état, ça ne passera pas la "CI".

Pour éviter de créer des liens morts, ce qui finirait inévitablement par arriver en l'absence de tests, j'ai ajouté une vérification dans une github action qui tourne à chaque commit sur `main`
Cette règle vérifie que ce qu'il y a dans `link`, correspond bien à un titre dans un des fichiers markdown. La règle prend comme format
`"link": "/voie-lyonnaise-2/#fontaines-sur-saône-à-lîle-barbe"`

(il faut remarquer le `/`après le `2`)

Bon en pratique, ça fonctionne très bien sans ce `/`, je peux donc tout à fait modifier la règle dans la CI, et éditer la VL1 :)

Reply:
J'ai bêtement fait un copier-coller de l'url depuis la page en question. Mais ok, je vais corriger pour que ce soit comme sur la VL1

Reply:
non non, ta méthode est probablement plus facile à implémentée. Je ne me souviens plus exactement, mais j'ai du rajouter le `/` à la main, ce qui ne facilite rien.
C'est plutôt à moi de changer la VL1 du coup :)